{
 "bad": [
  "django.apps.apps.ready",
  "django.apps.apps.set_installed_apps",
  "django.conf.settings.DATABASES",
  "django.conf.settings.DATABASES.update",
  "django.conf.settings.INSTALLED_APPS",
  "django.conf.settings.LANGUAGE_CODE",
  "django.conf.settings.ROOT_URLCONF",
  "django.conf.settings._setup",
  "django.conf.settings._wrapped",
  "django.core.management.commands.syncdb.connections",
  "django.db.backend",
  "django.db.backends.connections",
  "django.db.backends.mysql.base.connections",
  "django.db.connection.settings_dict",
  "django.db.models.connections",
  "django.db.models.sql.compiler.connections",
  "django.template.base.get_templatetags_modules.cache_clear",
  "django.template.base.templatetags_modules",
  "django.template.context._standard_context_processors",
  "django.template.engines.__dict__.pop",
  "django.template.engines._engines",
  "django.template.engines._templates",
  "django.template.loader.template_source_loaders",
  "finalizer",
  "logging.handlers.SysLogHandler.LOG_LOCAL7",
  "logging.root.handlers",
  "os.chmod",
  "os.mkdir",
  "os.path.exists",
  "os.unlink",
  "re.MULTILINE",
  "socket.SOCK_RAW",
  "socket.SOCK_STREAM",
  "socket.error",
  "subprocess32.CalledProcessError",
  "subprocess32.PIPE",
  "subprocess32.Popen",
  "subprocess32.TimeoutExpired",
  "warnings.warn",
  "zc.lockfile"
 ],
 "deps": {
  "_pytest": [
   "pytest/conda-forge/linux-64/pytest-3.7.0-py27_0",
   "pytest/conda-forge/linux-64/pytest-4.6.0-py37_0",
   "pytest/conda-forge/linux-64/pytest-6.0.2-py37hc8dfbb8_0"
  ],
  "contextlib": [
   "python/conda-forge/linux-64/python-3.6.10-he5300dc_1011_cpython",
   "python/conda-forge/linux-64/python-3.8.5-h1103e12_6_cpython"
  ],
  "distutils": [
   "python/conda-forge/linux-64/python-3.6.10-he5300dc_1011_cpython",
   "python/conda-forge/linux-64/python-3.8.5-h1103e12_6_cpython"
  ],
  "django": [
   "django/conda-forge/linux-64/django-1.8.14-py34_0",
   "django/conda-forge/noarch/django-1.8.19-py_0"
  ],
  "imp": [
   "python/conda-forge/linux-64/python-3.6.10-he5300dc_1011_cpython",
   "python/conda-forge/linux-64/python-3.8.5-h1103e12_6_cpython"
  ],
  "importlib": [
   "importlib/conda-forge/linux-64/importlib-1.0.4-py26_0",
   "python/conda-forge/linux-64/python-3.6.10-he5300dc_1011_cpython",
   "python/conda-forge/linux-64/python-3.8.5-h1103e12_6_cpython"
  ],
  "json": [
   "python/conda-forge/linux-64/python-3.6.10-he5300dc_1011_cpython",
   "python/conda-forge/linux-64/python-3.8.5-h1103e12_6_cpython"
  ],
  "logging": [
   "python/conda-forge/linux-64/python-3.6.10-he5300dc_1011_cpython",
   "python/conda-forge/linux-64/python-3.8.5-h1103e12_6_cpython"
  ],
  "ntpath": [
   "python/conda-forge/linux-64/python-3.6.10-he5300dc_1011_cpython",
   "python/conda-forge/linux-64/python-3.8.5-h1103e12_6_cpython"
  ],
  "os": [
   "python/conda-forge/linux-64/python-3.6.10-he5300dc_1011_cpython",
   "python/conda-forge/linux-64/python-3.8.5-h1103e12_6_cpython"
  ],
  "psutil": [
   "psutil/conda-forge/linux-64/psutil-5.7.3-py39h38d8fee_0"
  ],
  "pytest": [
   "pytest/conda-forge/linux-64/pytest-3.7.0-py27_0",
   "pytest/conda-forge/linux-64/pytest-4.6.0-py37_0",
   "pytest/conda-forge/linux-64/pytest-6.0.2-py37hc8dfbb8_0"
  ],
  "random": [
   "python/conda-forge/linux-64/python-3.6.10-he5300dc_1011_cpython",
   "python/conda-forge/linux-64/python-3.8.5-h1103e12_6_cpython"
  ],
  "re": [
   "python/conda-forge/linux-64/python-3.6.10-he5300dc_1011_cpython",
   "python/conda-forge/linux-64/python-3.8.5-h1103e12_6_cpython"
  ],
  "shutil": [
   "python/conda-forge/linux-64/python-3.6.10-he5300dc_1011_cpython",
   "python/conda-forge/linux-64/python-3.8.5-h1103e12_6_cpython"
  ],
  "socket": [
   "python/conda-forge/linux-64/python-3.6.10-he5300dc_1011_cpython",
   "python/conda-forge/linux-64/python-3.8.5-h1103e12_6_cpython"
  ],
  "subprocess": [
   "python/conda-forge/linux-64/python-3.6.10-he5300dc_1011_cpython",
   "python/conda-forge/linux-64/python-3.8.5-h1103e12_6_cpython"
  ],
  "subprocess32": [
   "subprocess32/conda-forge/noarch/subprocess32-3.5.4-py_1"
  ],
  "uuid": [
   "python/conda-forge/linux-64/python-3.6.10-he5300dc_1011_cpython",
   "python/conda-forge/linux-64/python-3.8.5-h1103e12_6_cpython"
  ],
  "warnings": [
   "python/conda-forge/linux-64/python-3.6.10-he5300dc_1011_cpython",
   "python/conda-forge/linux-64/python-3.8.5-h1103e12_6_cpython"
  ]
 }
}